components/dashboard-layout.tsx

```
@@ -45,6 +47,25 @@ export function DashboardLayout({
     return () => window.removeEventListener("scroll", handleScroll);
   }, []);
 
+  // Get username from userService
+  useEffect(() => {
+    // For demo, we'll use user with ID 2 (a non-admin)
+    const userData = userService.getById(2);
+    if (userData && userData.username) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)